blueoil/templates/lmnet/classification.tpl.py

```diff
@@ -50,7 +50,9 @@
 DATA_FORMAT = "NHWC"
 TASK = Tasks.CLASSIFICATION
 # In order to get instance property `classes`, instantiate DATASET_CLASS.
-CLASSES = DATASET_CLASS(subset="train", batch_size=1).classes
+dataset_obj = DATASET_CLASS(subset="train", batch_size=1)
+CLASSES = dataset_obj.classes
+step_per_epoch = int(dataset_obj.num_per_epoch/BATCH_SIZE)
```

Comment:
@lm-konda 
In the case
* dataset_obj.num_per_epoch is 15
* BATCH_SIZE is 10
* MAX_EPOCHS is 100
* lr settibng: tune2

I got this results `'boundaries': [33, 66, 99]`.
But It should be `[49 (=15/10*99*1/3), 99 (=15/10*99*2/3), 148 (=15/10*99)]`

Maybe I think to remove `int()`
```
step_per_epoch = float(dataset_obj.num_per_epoch)/BATCH_SIZE
```
instead of current 
https://github.com/blue-oil/blueoil/blob/0283ddd1324d0996ce8cc09ee85c435f5a196e2d/blueoil/templates/lmnet/classification.tpl.py#L55
is correct.
How about this ?

Reply:
Thanks! You are right. I removed int().